web/src/app/services/ServiceLabelList.js

```diff
@@ -38,29 +38,18 @@ const sortItems = (a, b) => {
   return 0
 }
 
-@withStyles(styles)
-export default class ServiceLabelList extends React.PureComponent {
-  static propTypes = {
-    serviceID: p.string.isRequired,
-  }
+const useStyles = makeStyles(styles)
 
-  state = {
+export default function ServiceLabelList(props) {
+  const [value, setValue] = useState({
```

Comment:
All of these can be broken into separate state values as opposed to one object.
`const [create, setCreate] = useState(false)`
`const [editKey, setEditKey] = useState(null)`
`const [deleteKey, setDeleteKey] = useState(null)`